app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,32 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <title>WorkYouCouldDo</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
-  <%= csrf_meta_tags %>
-</head>
-<body>
-
-<%= yield %>
-
-</body>
-</html>
+<% content_for :head do %>
+<%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+<% end %>
+
+<% content_for :page_title do %>
+  Work you could do
+<% end %>
+
+<% content_for :cookie_message do %>
+  <p>We uses cookies to make the site simpler.
+  <a href="/cookies">Find out more about cookies</a></p>
+<% end %>
+
+<% content_for :header_class do %>with-proposition<% end %>
+
+<% content_for :proposition_header do %>
+<div class="header-proposition">
+  <div class="content">
+    <div id="proposition-menu">
+      <h1 id="proposition-name">Work you could do</h1>
+    </div>
+  </div>
+</div>
+<% end %>
+
+<% content_for :content do %>
+  <main id="content">
+    <%= yield(:application_content) %>
```

Comment:
If we make this 
```ruby
content_for?(:application_content) ? yield(:application_content) : yield
```
then views that use this layout don't need to explicitly wrap their content in a `content_for(:application_content)`